src/test/java/org/apache/datasketches/memory/test/NioBitsTest.java

```diff
@@ -37,10 +37,11 @@ public void checkVMParams() {
   }
   
   @Test
-  //testing this beyond 2GB may not work on JVMs < 8GB.
-  //This must be checked manually
   public void checkGetAtomicFields() {
-    long cap = 1024L + Integer.MAX_VALUE;
+    //testing this beyond 2GB may not work on JVMs < 8GB.
+    //This should be checked manually
+   // long cap = 1024L + Integer.MAX_VALUE;
+    long cap = 1L << 10;
```

Comment:
I'll remove the @Ignore on this test once this is merged into the multi-module branch.

Reply:
Good idea.  Kind of a dumb test as it is now anyway.